src/panels/lovelace/common/coerce-entities-to-objects.js

```diff
@@ -0,0 +1,16 @@
+/** Coerce plain entity_id strings into {entity: 'entity_id'} objects **/
+export default function coerceEntitiesToObjects(entities) {
+  if (!Array.isArray(entities))
+    return entities;
+
+  const result = [];
+  entities.forEach(entity => {
+    if (typeof entity === 'string' || entity instanceof String) {
```

Comment:
This check is the same check twice?

Reply:
the 2nd is for `x = new String("test")` but Imo we don't need this

Reply:
Yeah, it's probably overkill here. I'll remove it